pkgs/development/libraries/raylib/default.nix

```diff
@@ -19,17 +20,24 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ cmake ];
-  buildInputs = [
-    mesa libGLU glfw libX11 libXi libXcursor libXrandr libXinerama
-  ] ++ lib.optional alsaSupport alsa-lib
-    ++ lib.optional pulseSupport libpulseaudio;
+  buildInputs =
+    if waylandSupport
+    then [ ]
```

Comment:
I have a question about this one. This way it seems, that if I enable `waylandSupport`, that I loose X11 support. Is this really the case with raylib? Can't I not have a build that supports both? Because if yes, then the only thing that should change for the buildinputs would be glfw.

Reply:
Also, if this means that I can build raylib without X11 support, then it would be great if we eventually make the X11 support optional too. But this can also be done in another PR.